cmd/dockerregistry/main.go

```diff
@@ -16,6 +18,18 @@ import (
 )
 
 func main() {
+	// Start the profiling server, if requested.
+	profilingAddr := os.Getenv("REGISTRY_PROFILING_ADDR")
```

Comment:
I've got this all working again as suggested, but one problem I'm having is the other usages of the webserver are hard-coded to bind the pprof server on loopback. When I use that and then expose the port with a service, I'm unable to connect via the service IP/port. If I change from loopback to binding on all interfaces (e.g. ":6060"), the service IP works.

This is using the all-in-one. Is something wrong with my local networking config, or should the pprof HTTP server be binding to all interfaces?